src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/config/services.yml

```diff
@@ -88,6 +88,13 @@ services:
         tags:
             - { name: tactician.handler, command: Surfnet\ServiceProviderDashboard\Application\Command\Entity\PublishEntityTestCommand }
 
+    Surfnet\ServiceProviderDashboard\Application\CommandHandler\Entity\EntityChangeRequestCommandHandler:
+        arguments:
+            $repository: '@Surfnet\ServiceProviderDashboard\Infrastructure\Manage\Client\EntityChangeRequestClient'
+            $issueType: 'TODO create configurable parameter'
```

Comment:
Is this something you should do, or just the value of the $issueType??

Reply:
This was a temporary reminder to change this to the actual issueType. In #523 this is fixed permanently.

_Background information_
I yanked code from the `PublishEntityProductionCommandHandler`. As the handling of this command is performed in a very similar fashion. For the time being, I left the Jira implementation of this command handler unchanged. As that was to be addressed in a successive story (see #523).

Given your previous comment. I'm planning on extracting some of the shared logic from these two command handlers, and put them in a service.